setup.py

```diff
@@ -84,24 +84,25 @@ def merge_req_lists(*req_lists: List[str]) -> List[str]:
 ]
 
 otl_dependencies = [
-    "opentelemetry-api==1.17.0",
-    "opentelemetry-exporter-otlp==1.17.0",
-    "opentelemetry-exporter-otlp-proto-grpc==1.17.0",
-    "opentelemetry-exporter-otlp-proto-http==1.17.0",
-    "opentelemetry-instrumentation==0.38b0",
-    "opentelemetry-proto==1.17.0",
-    "opentelemetry-sdk==1.17.0",
-    "opentelemetry-semantic-conventions==0.38b0",
+    "opentelemetry-api>=1.20.0",  # latest api required
+    "opentelemetry-exporter-otlp>=1.20.0",  # latest api required
+    "opentelemetry-exporter-otlp-proto-grpc>=1.20.0",  # latest api required
+    "opentelemetry-exporter-otlp-proto-http>=1.20.0",  # latest api required
+    "opentelemetry-instrumentation>=0.38b0",
+    "opentelemetry-proto>=1.20.0",  # latest api required
+    "opentelemetry-sdk>=1.20.0",  # latest api required
+    "opentelemetry-semantic-conventions>=0.41b0",
 ]
 
 stats_dependencies = merge_req_lists(
     _sql_dependencies,
-    requests_requirements,
     otl_dependencies,
     [
-        "wrapt==1.15.0",
-        "tqdm==4.62.3",
-        "omegaconf>=2.2.2",
+        "protobuf==3.20.3",  # required by ydb & otlp
```

Comment:
I don't understand why this has to be frozen. Looking at the dep tree it seems like there are no conflicts.
```
> pipdeptree --reverse --packages protobuf
protobuf==3.20.3
├── googleapis-common-protos==1.60.0 [requires: protobuf>=3.19.5,<5.0.0.dev0,!=4.21.5,!=4.21.4,!=4.21.3,!=4.21.2,!=4.21.1,!=3.20.1,!=3.20.0]
│   ├── opentelemetry-exporter-otlp-proto-grpc==1.20.0 [requires: googleapis-common-protos~=1.52]
│   │   └── opentelemetry-exporter-otlp==1.20.0 [requires: opentelemetry-exporter-otlp-proto-grpc==1.20.0]
│   └── opentelemetry-exporter-otlp-proto-http==1.20.0 [requires: googleapis-common-protos~=1.52]
│       └── opentelemetry-exporter-otlp==1.20.0 [requires: opentelemetry-exporter-otlp-proto-http==1.20.0]
├── opentelemetry-proto==1.20.0 [requires: protobuf>=3.19,<5.0]
│   ├── opentelemetry-exporter-otlp-proto-common==1.20.0 [requires: opentelemetry-proto==1.20.0]
│   │   ├── opentelemetry-exporter-otlp-proto-grpc==1.20.0 [requires: opentelemetry-exporter-otlp-proto-common==1.20.0]
│   │   │   └── opentelemetry-exporter-otlp==1.20.0 [requires: opentelemetry-exporter-otlp-proto-grpc==1.20.0]
│   │   └── opentelemetry-exporter-otlp-proto-http==1.20.0 [requires: opentelemetry-exporter-otlp-proto-common==1.20.0]
│   │       └── opentelemetry-exporter-otlp==1.20.0 [requires: opentelemetry-exporter-otlp-proto-http==1.20.0]
│   ├── opentelemetry-exporter-otlp-proto-grpc==1.20.0 [requires: opentelemetry-proto==1.20.0]
│   │   └── opentelemetry-exporter-otlp==1.20.0 [requires: opentelemetry-exporter-otlp-proto-grpc==1.20.0]
│   └── opentelemetry-exporter-otlp-proto-http==1.20.0 [requires: opentelemetry-proto==1.20.0]
│       └── opentelemetry-exporter-otlp==1.20.0 [requires: opentelemetry-exporter-otlp-proto-http==1.20.0]
├── streamlit==1.26.0 [requires: protobuf>=3.20,<5]
│   └── streamlit-chat==0.1.1 [requires: streamlit>=0.63]
└── ydb==3.3.6 [requires: protobuf>=3.13.0,<5.0.0]
```

Reply:
Updated